metaflow/plugins/kfp/kfp.py

```diff
@@ -1244,9 +1042,18 @@ def _create_exit_handler_op(self) -> ContainerOp:
         if self.notify_on_success:
             notify_variables["METAFLOW_NOTIFY_ON_SUCCESS"] = self.notify_on_success
 
-        return exit_handler(
-            flow_name=self.name,
-            status="{{workflow.status}}",
-            kfp_run_id=dsl.RUN_ID_PLACEHOLDER,
-            notify_variables=notify_variables,
-        )
+        exit_handler_command = [
+            "bash",
+            "-ec",
+            " python /opt/metaflow/metaflow_container_op/kfp_exit_handler.py "
```

Comment:
Guessing the `workflow_spec_after` [linked](https://drive.google.com/file/d/1t7znhdV2WuLF_6t9WH1a583TZ4gckn92/view) in the description is just outdated, as that seems to still have the full `exit_handler` inlined? So guessing the reduction will be even better with this change?